src/Sulu/Bundle/AdminBundle/Resources/js/components/ArrowMenu/item.scss

```diff
@@ -2,24 +2,35 @@
 
 $itemColor: $black;
 $itemColorActive: $blueZodiac;
+$disabledColor: $silver;
 
 .item {
+    background: none;
+    border: none;
+    display: block;
     height: 22px;
     color: $itemColor;
     font-size: 12px;
     line-height: 22px;
     margin-bottom: 6px;
     cursor: pointer;
+    text-align: start;
+    width: 100%;
 
     &:last-child {
         margin-bottom: 0;
     }
 
     &.active,
-    &:hover {
+    &:hover:not([disabled]) {
```

Comment:
What do the squared brackets mean? Have never seen that, and a google search didn't yield any results... Are you sure that should not be `&:hover:not(:disabled)`?